src/Appointments.Console/Appointment Management System/Repositories/AppointmentRepository.cs

```diff
@@ -0,0 +1,165 @@
+﻿using Appointment_Management_System.Interfaces;
+using Appointment_Management_System.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Appointment_Management_System.Repositories
+{
+    public class AppointmentRepository: IManagementRepository<Appointment>
+    {
+        public void Create( List<Appointment> appointments, List<Customer> customers)
+        {
+            Console.WriteLine("Create Appointment");
```

Comment:
Violation of single responsibility, the repository manages persistency and user interface